### ISSUE TITLE
Seems mistake with one of the JOIN example

### ISSUE THREAD

**Issue by rbereziuk** [btholt/complete-intro-to-sqlite#5]
![2024-11-12_20-16](https://github.com/user-attachments/assets/1a8f2236-71a1-4e96-b620-001718655484)

https://sqlite.holt.courses/lessons/intermediate-sql/other-types-of-joins

1. As I understood from (1) the next query should be to select ONLY Artists that don't have an Album. But query (2) selects Artists with and without Album. In video on the screen display only Artists without Album, but if scroll up, also present Artists with Album. Seems in query missing `WHERE A.ArtistId IS NULL;` part.

2. Also for me, strange sentence (3) in context of (1) and (2) above. OUTER verb don't add any functionality to query, and we could omit that. But in (3) might get the impression that this verb have effect to filter Artists with albums.

**Comment by Xavier-Brinon** [comment]
I just passed this part of the course and noted the very same thing !

**Comment by dtauer** [comment]
Yes, you are correct! Nice catch (both of you!). I'll update the site.
